google-cloud-clients/google-cloud-bigtable/src/main/java/com/google/cloud/bigtable/data/v2/BigtableDataSettings.java

```diff
@@ -43,12 +43,25 @@
 /**
  * Settings class to configure an instance of {@link BigtableDataClient}.
  *
- * <p>Sane defaults are provided for most settings:
+ * <p>Some defaults are provided for most settings:
```

Comment:
Since you have made this change here, could you change line 242 too to "some defaults".  " Initializes a new Builder with **sane** defaults for all settings."